polyfills/NodeList/prototype/@@iterator/config.json

```diff
@@ -0,0 +1,19 @@
+{
+	"browsers": {
+		"ie": "*",
+		"safari": "<9",
+		"chrome": "< 50",
+		"firefox": "< 40",
+		"ios_saf": "*",
+		"opera": "< 50"
+	},
+	"dependencies": [
+		"Symbol.iterator"
+	],
+	"license": "MIT",
+	"spec": "https://dom.spec.whatwg.org/#nodelist",
+	"repo": "https://github.com/medikoo/es6-iterator",
+	"build": {
+		"commonjs": true
+	}
```

Comment:
You'll also need to declare the ECMAScript version of the polyfill as you are using `const`.
If you add `"esversion": 6` to this file then the service will run the polyfill through babel before creating the bundle for the browser.